src/components/MusicMarkdownNavbar.js

```diff
@@ -105,34 +121,16 @@ class MusicMarkdownNavbar extends React.Component {
             </Popper>
           </Toolbar>
         </AppBar>
-        <Drawer open={toolbarOpen} variant={'persistent'} anchor={'bottom'}>
-          <Divider />
-          <MusicToolbar></MusicToolbar>
-        </Drawer>
-      </>
-    );
-
-    const BaseNavbar = () => (
-      <>
-        <AppBar position={'sticky'} key='top-navbar'>
-          <Toolbar>
-            <Button className={classes.reactRouterHoverInherit} component={Link} to='/'>
-              <img src="music-markdown.svg" width={50} alt="Music Markdown" />
-            </Button>
-            <RepositoriesNavDropdown {...this.props} />
-            <div className={classes.grow} />
-          </Toolbar>
-        </AppBar>
       </>
     );
 
     const basePath = this.props.location.pathname.split('/')[1];
 
     switch (basePath) {
     case 'repos':
-      return MusicMarkdownNavbar();
+      return BaseNavbar(true);
```

Comment:
This would have the music toolbar show up in repo navigation as well. We should only show it in `/viewer` or `/editor`.